application/helpers/admin/import_helper.php

```diff
@@ -738,10 +739,16 @@ function XMLImportQuestion($sFullFilePath, $iNewSID, $iNewGID, $options = array(
                 if (!in_array($insertdata['language'], $aLanguagesSupported)) {
                     continue;
                 }
+
+                // Avoid inserting attribute if the language of the question is not in the survey.
```

Comment:
We already have a continue before , why not fix $aLanguagesSupported directly ?
https://github.com/LimeSurvey/LimeSurvey/blob/d4365162261f1867a3895bb83232da764e22788b/application/helpers/admin/import_helper.php#L645